app/controllers/medicaid/health_recent_pregnancy_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class HealthRecentPregnancyController < Medicaid::ManyMemberStepsController
+    private
+
+    def skip?
+      no_pregnancies? ||
+        only_males? ||
+        (single_female_member_is_pregnant? && update_primary_female_member)
```

Comment:
the way we've been handling this in other "multi member" controllers is:

- the single member us updated via the "yes/no" controller
-  we skip the follow on controller for single member households. 

thoughts on the benefits and drawbacks of that approach vs this one?